src/components/dialogs/create-study-dialog/create-study-dialog-utils.js

```diff
@@ -34,4 +36,5 @@ export const createStudyDialogFormValidationSchema = yup.object().shape({
     [CASE_UUID]: yup.string().required(),
     [CASE_FILE]: yup.mixed().nullable().required(),
     [DIRECTORY]: yup.string().required(),
+    [CASE_FORMAT]: yup.string().optional(),
```

Comment:
I don't think we use optional on string in our projects, why adding it as we use empty string ?
https://github.com/jquense/yup#schemaoptional-schema

Reply:
Why do you want the field to be optional? Are there cases where we can be without the format? I believe it should be required

Reply:
i changed it to be required